accname/name/comp_content_visibility.tentative.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+	<head>
+	  <title>Name Comp: content visibility (Tentative)</title>
+	  <script src="/resources/testharness.js"></script>
+	  <script src="/resources/testharnessreport.js"></script>
+	  <script src="/resources/testdriver.js"></script>
+	  <script src="/resources/testdriver-vendor.js"></script>
+	  <script src="/resources/testdriver-actions.js"></script>
+	  <script src="/wai-aria/scripts/aria-utils.js"></script>
+	</head>
+	<body>
+
+		<h1>Testing accessibility of content-visibility: auto</h1>
+
+	  <h2>Accname</h2>
+	  
+	  <div style="height:5px; overflow:scroll;">
+		  <div>
+		  	<p>Some content</p>
+		  	<p>Some content</p>
+		  	<p>Some content</p>
+		  	<p>Some content</p>
+		  	<p>Some content</p>
+		  </div>
+		  <div style="content-visibility:auto"> 
+		  	<h2 class="el">test heading</h2>
+		    <button class="el">test button</button>
+		    <a class="el" href="#">test link</a>
+		  </div>
+	  </div>
+		
+		<script>
+		  const els = document.querySelectorAll('.el');
+		  if (!els.length) {
+		    throw `No`;
+		  }
+		  for (const el of els) {
+		    promise_test(async t => {
+		      let computedLabel = await test_driver.get_computed_label(el);
+		      let computedRole = await test_driver.get_computed_role(el);
+
+		      console.log('el-text-content: '+el.textContent+'; comp-role-f: '+computedRole+'; comp-label-f: '+computedLabel);
```

Comment:
You can still use the existing aria utils logging to abstract this... Just leave the expected role and labels as something no engine will ever use, like data-expectedrole="SPEC_AMBIGUOUS_LOG_VALUE" and all engines will log a message like ~"Role for for element was `group`; Expected: SPEC_AMBIGUOUS_LOG_VALUE"

Reply:
Is it acceptable for all engines to fail, even if the outcome is uncertain?

Reply:
In a tentative test that we are leveraging to determine the correct behavior, yes! Absolutely. Tentative tests don't get counted in the total WPT metrics, so there is no downside. 

The much more concerning downside to putting in an expectation for a potentially ambiguous spec, is that the implementations may make a change based on the tentative test. That would be bad, because if the WG later determines the expectation to be invalid, now the engine would have to roll back the change and it causes a webcompat issue.